pkg/resource/model_package_group/sdk.go

```diff
@@ -17,8 +17,6 @@ package model_package_group
 
 import (
 	"context"
-	"strings"
```

Comment:
revert?, curious why we see these changes only on your PR and why now

Reply:
after adding the go fmt etc to develop unit tests everytime I ran gofmt it does this. I can revert it however